README.md

```diff
@@ -6,24 +6,27 @@
 
 :balloon: Welcome to Conduit! :wave:
 
-Conduit is an ultralight service mesh for Kubernetes from the makers of [Linkerd][l5d]. It
-features a native proxy, written in [Rust][rust], that boasts the performance of C without
-all the heartbleed.
+Conduit is an ultralight service mesh for Kubernetes. It features a minimalist
+control plane written in Go, and a native proxy data plane written in
+[Rust][rust] that boasts the performance of C without the heartbleed.
 
-Conduit is **experimental**. It is capable of proxying all TCP traffic, and reporting
+Conduit is **alpha**. It is capable of proxying all TCP traffic, and reporting
 top-line metrics (success rates, latencies, etc) for all HTTP, HTTP/2, and gRPC traffic.
 It currently does not work with websockets or with HTTP tunneling--see the
 `--skip-outbound-ports` flag for how to exclude these types of traffic.
 
-Follow our progress towards production-readiness here and on [Twitter][twitter].
+## Join the fun
```

Comment:
Maybe "Get involved"?  Don't really think "join the fun" fits with the rest of the doc